xslt.c

```diff
@@ -136,3 +136,12 @@ int xslt(const char *xsl, const char *xml, char **xml_txt,
 
   return ok;
 }
+
+/*
+ * Function: init_exslt
+ * ----------------------------
+ *  Calls exsltRegisterAll() to enable exsl namespace at templates
+ */
+void init_exslt() {
+  exsltRegisterAll();
```

Comment:
We'll need the exslt headers.  Can you add the following with the other include directives:
`#include <libexslt/exslt.h>`